### PR TITLE
chore(updated supported releases): Updated support for ParrotOS 6.1

### PR DESCRIPTION
**Pull request by EuroNutellaMan**
Added support for:

- ParrotOS 6.1

Removed support for:

- ParrotOS 5.9

### CODE REVIEW

File: quickget [hunk 1]
```diff
@@ -942,7 +942,7 @@ function releases_oraclelinux() {
 }
 
 function releases_parrotsec() {
-    echo 6.0 5.3
+    echo 6.1 6.0
```
Comment (lj3954) [inline quickget:945]:
```suggestion
    echo $(web_pipe "https://download.parrot.sh/parrot/iso/" | grep -o -P '(?<=href=")[0-9].*(?=/")' | sort -nr | head -n +2)
```

**Review by lj3954 — CHANGES_REQUESTED** [review]
I've suggested changes to allow quickget to dynamically find the latest 2 releases. 

In addition, please remove NixOS from the PR, #1266 includes changes for that operating system.

**Comment by EuroNutellaMan** [comment]
I should have removed the NixOS changes

**Comment by EuroNutellaMan** [comment]
I can't figure out why the lint shellcheck keeps failing (not too familiar with github yet), do you want me to close this PR and redo it with the suggested changes?

I assumed it was due to the 2nd commit in which I accepted your suggestion, which isn't properly formatted but for whatever reason it doesn't let me rename that commit.

**Comment by lj3954** [comment]
> I can't figure out why the lint shellcheck keeps failing (not too familiar with github yet), do you want me to close this PR and redo it with the suggested changes?
> 
> I assumed it was due to the 2nd commit in which I accepted your suggestion, which isn't properly formatted but for whatever reason it doesn't let me rename that commit.

It's because some of ShellCheck's lints don't align with the formatting used by quickemu. Please add this line above.

#shellcheck disable=SC2046,SC2005

**Comment by EuroNutellaMan** [comment]
> > I can't figure out why the lint shellcheck keeps failing (not too familiar with github yet), do you want me to close this PR and redo it with the suggested changes?
> > 
> > I assumed it was due to the 2nd commit in which I accepted your suggestion, which isn't properly formatted but for whatever reason it doesn't let me rename that commit.
> 
> It's because some of ShellCheck's lints don't align with the formatting used by quickemu. Please add this line above.
> 
> #shellcheck disable=SC2046,SC2005

Where should I add it exactly?

**Review by flexiondotorg — APPROVED** [review]

**Pull request merged**